lib/rest-adapter.js

```diff
@@ -403,6 +411,14 @@ RestAdapter.prototype._registerMethodRouteHandlers = function(router,
     // Express 4.x only supports delete
     verb = 'delete';
   }
+
+  //For now: exclude paths with parameters like :id
+  //and handler functions with err obj i.e. (err, req,res,next)
+  if (!route.path.includes(':') && handler.length < 4) {
+    handler.methodName = sharedMethod.name;
+    handler.path = route.fullPath;
+    customRouter.registerPathAndHandlers({ verb: verb, fullPath: route.fullPath }, handler);
+  }
   router[verb](route.path, handler);
```

Comment:
@bajtos: In our case, when we have one handler for verb: `all` and another handler for verb `get`, our router will register both, and will invoke `get` one on `GET` request. 
So, the question is: do we drop `get` handlers when `all` is present or chain them such as:  `all:handler` executes -> `get:handler` executes (if this' the case, then we'll have multiple, at least two, handlers to execute)

please advise. Thanks.